public/components/facet/facet.js

```diff
@@ -90,6 +98,7 @@ export default ['$http', function($http) {
                 d.selected = true;
                 if (item.negative) {
                   d.negative = true;
+                  d.count = 99999;
```

Comment:
This is kind of a hack, since I want negative fields to show up first, and fields are sorted by # of entries descending.